app/views/annotations/_annotation_modal.html.erb

```diff
@@ -47,9 +47,12 @@
           <%= t('annotations.default_deduction') %>
         </p>
       <% end %>
+      <% if change_one_option %>
+        <br><br><%= f.check_box :change_one, id: 'change_one_annotation' %> <%= title_one %>
```

Comment:
A couple of things here:

1. Wrap the input and text label in a `p` tag rather than using `<br>`
2. Use an `f.label` to explicitly link the text to the checkbox input.
3. Don't pass `title_one` as a local, just call `t('annotations.update_one')` right here
4. Also, let's reverse the meaning of this checkbox. Call it something like "Change all existing uses of this annotation text." (I guess you can change the name from `change_one` to `change_all`.)